### PR TITLE
Airflow network distribution outputs for node pressure and flow now report zero when the distribution system is off

### PR DESCRIPTION
**Pull request by lgu1234**
## Pull request overview

Addresses #5687.
At times when the airloop is not operating, the node pressures and duct airflows are reported as zero in airflow network distribution outputs.

Pressure differences were found in AirflowNetwork_MultiZone_House.idf and AirflowNetwork_MultiZone_House_OvercoolDehumid.idf.

Airflow rate differences were found in AirflowNetwork_PressureControl.idf and AirflowNetwork_Simple_SmallOffice.idf.

The above differences are expected.

Unit test: No new unit test function is created. Addition zero value test is added in the existing function: EnergyPlusFixture, TestAFNPressureStat in AirflowNetworkBalanceManager.unit.cc. 
### Work Checklist

Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
- [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
- [x] At least one of the following appropriate labels must be added to this PR to be consumed into the changelog:
  - Defect: This pull request repairs a github defect issue.  The github issue should be referenced in the PR description
  - Refactoring: This pull request includes code changes that don't change the functionality of the program, just perform refactoring
  - NewFeature: This pull request includes code to add a new feature to EnergyPlus
  - Performance: This pull request includes code changes that are directed at improving the runtime performance of EnergyPlus
  - DoNoPublish: This pull request includes changes that shouldn't be included in the changelog
### Review Checklist

This will not be exhaustively relevant to every PR.
- [x] Code style (parentheses padding, variable names)
- [x] Functional code review (it has to work!)
- [x] If defect, results of running current develop vs this branch should exhibit the fix
- [x] CI status: all green or justified
- [x] Performance: CI Linux results include performance check -- verify this
- [x] Unit Test(s)
- C++ checks:
  - [x] Argument types
  - [x] If any virtual classes, ensure virtual destructor included, other things
- IDD changes: n/a

### CODE REVIEW

**Comment by nrel-bot-3** [comment]
@mjwitte @lgu1234 @lgentile it has been 14 days since this pull request was last updated.

**Comment by mjwitte** [comment]
@lgu1234 This fixes `AFN Linkage Node 1 to Node 2 Mass Flow Rate` and `AFN Node Total Pressure` but `AFN Linkage Node 1 to Node 2 Pressure Difference` is still reporting non-zero values.

File: src/EnergyPlus/AirflowNetworkBalanceManager.cc [hunk 1]
```diff
@@ -6732,7 +6732,20 @@ namespace AirflowNetworkBalanceManager {
...
+			}
+			for ( i = AirflowNetworkNumOfSurfaces + 1; i <= AirflowNetworkNumOfLinks; ++i ) {
+				AirflowNetworkLinkReport( i ).FLOW = 0.0;
+				AirflowNetworkLinkReport( i ).FLOW2 = 0.0;
+				AirflowNetworkLinkReport( i ).VolFLOW = 0.0;
+				AirflowNetworkLinkReport( i ).VolFLOW2 = 0.0;
+			}
+		}
+		
+		if ( !( AirflowNetworkFanActivated && SimulateAirflowNetwork > AirflowNetworkControlMultizone ) ) return;
```
Comment (mjwitte) [inline src/EnergyPlus/AirflowNetworkBalanceManager.cc:6749]: @lgu1234 Can this return be moved inside the new if block above?

**Comment by mjwitte** [comment]
@lgu1234 I update this branch with current develop, so be sure to pull it down before making changes.

File: src/EnergyPlus/AirflowNetworkBalanceManager.cc (hunk 1 above)
Comment (lgu1234) [inline src/EnergyPlus/AirflowNetworkBalanceManager.cc:6749]: @mjwitte Both "if" statements have different logics.

**Comment by lgu1234** [comment]
@mjwitte A line was added to set up the pressure different = 0 for an output variable: AFN Linkage Node 1 to Node 2 Pressure Difference, when the distribution system is off. When I add the output variable in AirflowNetwork_PressureControl.idd, differences were found as expected.

**Pull request merged**